var/spack/repos/builtin/packages/szx/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Szx(AutotoolsPackage):
+    """An ultra fast error bounded compressor for scientific datasets"""
+
+    homepage = "https://github.com/szcompressor/szx"
```

Comment:
I don't see an `szx` sub repository at "https://github.com/szcompressor/szx"

Reply:
@disheng222 when will szx be released?  I can make this private if it going to be a while.

Reply:
This work was partially supported by the fund from Aramco. We need to ask for their permission for release. Will keep you posted.